Log spam when opening file outside of Git repo

When you open a file not inside a Git repository, e.g. `~/.cache/helix/helix.log`, this is printed to Helix's log file:
```
2023-04-18T10:45:04.134 helix_vcs [ERROR] Error {
    context: "failed to open git repo",
    source: Discover(
        NoGitRepository {
            path: "/home/me/.cache/helix",
        },
    ),
}
2023-04-18T10:45:04.134 helix_vcs [ERROR] failed to open diff base for /home/me/.cache/helix/helix.log
2023-04-18T10:45:04.134 helix_vcs [ERROR] Error {
    context: "failed to open git repo",
    source: Discover(
        NoGitRepository {
            path: "/home/me/.cache/helix",
        },
    ),
}
2023-04-18T10:45:04.134 helix_vcs [ERROR] failed to obtain current head name for /home/me/.cache/helix/helix.log
```

I expect that nothing is printed to the logs.

The bug occurs on helix 23.03 and ca65d312e715413cd5675cafd599d74534900d3f.

--- a/helix-vcs/src/lib.rs
+++ b/helix-vcs/src/lib.rs
@@ -46,8 +46,8 @@ impl DiffProviderRegistry {
             .find_map(|provider| match provider.get_diff_base(file) {
                 Ok(res) => Some(res),
                 Err(err) => {
-                    log::error!("{err:#?}");
-                    log::error!("failed to open diff base for {}", file.display());
+                    log::info!("{err:#?}");
+                    log::info!("failed to open diff base for {}", file.display());
                     None
                 }
             })
@@ -59,8 +59,8 @@ impl DiffProviderRegistry {
             .find_map(|provider| match provider.get_current_head_name(file) {
                 Ok(res) => Some(res),
                 Err(err) => {
-                    log::error!("{err:#?}");
-                    log::error!("failed to obtain current head name for {}", file.display());
+                    log::info!("{err:#?}");
+                    log::info!("failed to obtain current head name for {}", file.display());
                     None
                 }
             })

